### PR TITLE
[Jetsnack] Add tests

### PR DESCRIPTION
**Pull request by manuelvicnt**
Adds simple navigation tests to Jetsnack.

Note: We don't have to worry about the animations in the bottom bar, the test framework takes care of that 👌

### CODE REVIEW

File: Jetsnack/app/src/androidTest/java/com/example/jetsnack/AppTest.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    @Test
+    fun app_canNavigateToAllScreens() {
+        // Check app launches at HOME
+        composeTestRule.onNodeWithText("HOME").assertIsDisplayed()
+        composeTestRule.onNodeWithText("Android's picks").assertIsDisplayed()
+
+        // Navigate to Search
+        composeTestRule.onNodeWithTag(HomeSectionTag.SEARCH.toString()).performClick()
```
Comment (JoseAlcerreca) [inline Jetsnack/app/src/androidTest/java/com/example/jetsnack/AppTest.kt:58]: This should be a last resort, did you try to match using hierarchy matchers?

Comment (manuelvicnt) [reply Jetsnack/app/src/androidTest/java/com/example/jetsnack/AppTest.kt:58]: Tell me more about that... do you have an example I could take a look at?

Comment (JoseAlcerreca) [reply Jetsnack/app/src/androidTest/java/com/example/jetsnack/AppTest.kt:58]: Hierarchical matchers or selectors: https://developer.android.com/jetpack/compose/testing

Even if you can't do this using a known element, you could add a single testTag for the bar and then use the hierarchical matchers to look for the specific buttons inside.

**Review by JoseAlcerreca — APPROVED** [review]

**Review by nickbutcher — APPROVED** [review]
Thanks!

**Pull request merged**